app/src/components/widgets/charts/investigatedTrendChart/common/constants.js

```diff
@@ -0,0 +1,481 @@
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-15': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-16': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-17': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-18': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-19': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-20': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-21': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-22': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-23': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-24': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-25': [
+      {
+        values: {
+          to_investigate: '41.79',
+          investigated: '58.21',
+        },
+      },
+    ],
+    '2018-09-26': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-27': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-28': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-29': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-09-30': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-10-01': [
+      {
+        values: {
+          to_investigate: '43.18',
+          investigated: '56.82',
+        },
+      },
+    ],
+    '2018-10-02': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-10-03': [
+      {
+        values: {
+          to_investigate: '0',
+          investigated: '0',
+        },
+      },
+    ],
+    '2018-10-04': [
+      {
+        values: {
+          to_investigate: '39.72',
+          investigated: '60.28',
+        },
+      },
+    ],
+  },
+};
+
+export const TEST_DATA = {
```

Comment:
It's better to move test data to another file

Reply:
moved